hips/hip-0012.md

```diff
@@ -0,0 +1,178 @@
+However, a "generalized" process that can span multiple major versions over multiple
+years is ambitious.
+So this proposal attempts to merely spell out a formal process for Helm 4.
+
+## Rationale
+
+Describe why particular design decisions were made.
+
+## Specification
+
+### The Release Engineer
+
+A single individual will oversee the planning and development phases of Helm 4.
+This person will be the _Release Engineer_.
+The person will have the following responsibilities:
+
+- Lead the kick-off meeting
+- Define the process for including features
+- Determine the timelines for development
+- Make the determination of when the project has indeed reached a milestone
+- Name and oversee releases
+- After the release, the release manager will determine what the "intent" of a Helm behavior was (which informs determining whether an incoming breaking change is a feature or a bugfix)
+
+#### Determining Who Will Be Release Engineer
+
+The release engineer must be a core maintainer on the `github.com/helm/helm` project.
+The engineer must not be in danger of losing core maintainership due to inactivity.
+Anyone can nominate a core maintainer to become the release engineer (self-nominations are encouraged).
+Any nominee must _agree_ to the nomination.
+The Helm project maintainers will determine which of the nominated maintainers shall be the release engineer.
+
+In the event that a release engineer cannot perform the duties required,
+the Helm project maintainers may choose a replacement.
+
+### The Process
+
+There will be three phases to Helm 4 development:
+
+1. Planning
+2. Feature development
+3. Release preparation
+
+The timeline will be approximately:
+
+- Naming of a Release Engineer: Sept. 2021
+- Kick-off meeting: Oct. 2021
+- Earliest that engineering can begin: Jan. 2022
+- Earliest that a release could possibly be: May, 2022
+
+More likely, the release will be late 2022 to midyear 2023.
+
+#### Planning
+
+This phase begins when the release engineer is named.
+The release engineer shall schedule a kick-off meeting in which all Helm maintainers (from any Helm org project) will be invited to participate.
+Beyond these participants, the release engineer may determine who else will be included, and may determine the meeting to be public.
+The kick-off meeting will open the floor to feature ideas and changes, with preference toward ideas already documented as HIPs.
+
+This process will continue as long as the release engineer determines necessary,
+but not less than three months.
+This minimum time frame is intended to give ample time for the community to provide input.
+
+All features and breaking changes MUST be described in a HIP, and that HIP must pass the usual process.
+This provides three benefits:
+
+- All changes are documented
+- All changes have had a period of scrutiny prior to the PR being written
+- There is a clear record of alternative considerations
```

Comment:
I think that this is important for the progress and community participation of Helm 4.  I would recommend mentioning that a project list/backlog of all HIPs are assigned to Helm 4 release prior to development starting. This list then provides the community with a blueprint of requirements for Helm 4 which helps feedback and  contribution more easily.